src/Component.tsx

```diff
@@ -304,8 +304,17 @@ export const Sheet = defineComponent({
       <Teleport to="body">
         <Transition
           css={false}
-          onEnter={handleAnimationEnter}
-          onLeave={handleAnimationLeave}
+          onEnter={
+            props.handleAnimationEnter !== undefined ?
+              props.handleAnimationEnter :
+              handleAnimationEnter 
+              
+          }
+          onLeave={
+            props.handleAnimationLeave !== undefined ?
+              props.handleAnimationLeave :
+              handleAnimationLeave
+          }
```

Comment:
Firstly thanks for the PR, We should treat null and undefined differently for this behavior.

If user passes null: no transition
If user passes undefined: use default transition callback
If user passes a function: use that function as a transition callback.

And I prefer ternary operators in new line :)

```suggestion
          onEnter={
            props.handleAnimationEnter === null
              ? undefined
               : props.handleAnimationEnter || handleAnimationEnter
          }
          onLeave={
            props.handleAnimationLeave === null
              ? undefined :
              props.handleAnimationLeave || handleAnimationLeave
          }
```